main.go

```diff
@@ -4,11 +4,28 @@ Copyright © 2024 NAME HERE <EMAIL ADDRESS>
 package main
 
 import (
+	"context"
+	"log"
+
 	"github.com/jbarzegar/ron-mod-manager/cmd"
+	"github.com/jbarzegar/ron-mod-manager/db"
 	stateManagement "github.com/jbarzegar/ron-mod-manager/state-management"
+	_ "github.com/mattn/go-sqlite3"
 )
 
 func main() {
+	client, err := db.InitClient()
+
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	defer client.Close()
```

Comment:
**nitpick:** I really prefer when Go is written in this format:

```suggestion
func main() {
	client, err := db.InitClient()
	if err != nil {
		log.Fatalln(err)
	}
	defer client.Close()
```

My reasoning behind this is it makes it clear what distinct logical groups of code are. That block interacts with client, checks the error, and defers closing. If I see a newline, my brain switches to new context. Highly opinionated, but I suggest at least putting the err check right belowl